lms/static/lms/js/spec/main.js

```diff
@@ -679,6 +679,7 @@
     });
 
     testFiles = [
+        'course_experience/js/spec/course_outline_factory_spec.js',
```

Comment:
I wonder if this could all be more discoverable.  Doesn't have to be part of this PR.